Feat/categories slider UI

KhalidElnaghi opened the pull request.

abdelrahman-essawy reviewed (changes requested):
![image](https://github.com/abdelrahman-essawy/a3maly-dashboard-client/assets/30594965/acbff567-32e0-44de-ad53-4b40c578a6f7)

1. the 2 arrows need to be aligned correctly
2. the width of categories wrapper chould have max width of the avilable space. 
3. remove the tiny border-radius, or combine the search and categories filter in the same component.

take a look at https://swiperjs.com/

abdelrahman-essawy reviewed (changes requested):
![image](https://github.com/abdelrahman-essawy/a3maly-dashboard-client/assets/30594965/75668405-f6e7-4422-a076-72b292ac8860)
almost perfect,
- left arrow is not working
- can we get the 2 arrows to be beside the words ?

KhalidElnaghi commented:
- The left arrow was not working because of the loop so I removed it 
- I changed
![Capture](https://github.com/abdelrahman-essawy/a3maly-dashboard-client/assets/46836231/dfeff5b5-bdd8-460b-8d8f-202f8df855ed)
 the arrow place as you mention
